### PR TITLE
vGrid

### LINKED ISSUE #308: Website consistency
- [x] Page-wide consistency when producing toasts to show input (so all or none on a given page) and 
MB component calls should list parameters vertically and alphabetically for reader clarity.

     - [ ] Anchor.razor
     - [ ] AnimatedNavigation.razor
     - [ ] AutocompleteTextField.razor
     - [ ] BladeSet.razor
     - [ ] Button.razor
     - [ ] Card.razor
     - [ ] CascadingDefaults.razor
     - [ ] Checkbox.razor
     - [ ] ChipsSelectMulti.razor
     - [ ] ChipsSelectSingle.razor
     - [ ] CircularProgress.razor
     - [ ] ConfirmationDialog.razor
     - [ ] DataTable.razor
     - [ ] DatePicker.razor
     - [ ] DebouncedTextField.razor
     - [ ] Dialog.razor
     - [ ] Divider.razor
     - [ ] Drawer.razor
     - [ ] FloatingActionButton.razor
     - [ ] FormValidation.razor
     - [ ] Grid.razor
     - [ ] Icon.razor
     - [ ] IconButton.razor
     - [ ] IconButtonToggle.razor
     - [ ] Index.razor
     - [ ] LinearProgress.razor
     - [ ] List.razor
     - [ ] Menu.razor
     - [ ] NumericDecimalField.razor
     - [ ] NumericDoubleField.razor
     - [ ] NumericIntField.razor
     - [ ] PagedDataList.razor
     - [ ] Paginator.razor
     - [ ] RadioButton.razor
     - [ ] RadioButtonGroup.razor
     - [ ] SegmentedButtonMulti.razor
     - [ ] SegmentedButtonSingle.razor
     - [ ] Select.razor
     - [ ] Shield.razor
     - [ ] Slider.razor
     - [ ] SlidingContent.razor
     - [ ] SlidingTabBar.razor
     - [ ] Snackbar.razor
     - [ ] Switch.razor
     - [ ] TabBar.razor
     - [ ] TextArea.razor
     - [ ] TextField.razor
     - [ ] Toast.razor
     - [ ] Tooltip.razor
     - [ ] TopAppBar.razor
     - [ ] Typography.razor

- [x] index.razor : 'Use <a href="https://www.nuget.org/packages/Material.Blazor/" target="_blank">nuget</a> to procure the latest release.'

### LINKED ISSUE #338: Implement Snackbars
Tentatively for MB v2.0.0

### LINKED ISSUE #346: MBDatePicker allows start/end dates that cause date underflow/overflow conditions
MBDatePicker needs to protect itself against date artithmetic errors by disallowing dates within a month of MinDate & MaxDate

### PR DESCRIPTION
**Pull request by MarkStega**

### CODE REVIEW

File: Material.Blazor.Website/Pages/Anchor.razor [hunk 1]
```diff
@@ -23,9 +23,11 @@
                         Anchor
                     </h2>
 
-                    <p>The Anchor is a non-visual component. If you use Toast and/or Tooltip components this
-                    must be placed in your app.razor or the layout page used for all application components
-                    that use toasts and tooltips.</p>
+                    <p>
+                        The Anchor is a non-visual component. If you use Toast and/or Tooltip components this
```
Comment (stefanloerwald) [inline Material.Blazor.Website/Pages/Anchor.razor]: Snackbar should be included here too.

File: Material.Blazor.Website/Pages/ChipsSelectMulti.razor [hunk 2]
```diff
@@ -59,10 +73,10 @@
         new MBIconBearingSelectElement<string> { SelectedValue = "brit-short", Label = "British Shorthair" },
         new MBIconBearingSelectElement<string> { SelectedValue = "russ-blue", Label = "Russian Blue", Icon="admin_panel_settings" },
         new MBIconBearingSelectElement<string> { SelectedValue = "ice-invis", Label = "Icelandic Invisible", Icon="card_membership" }
-    };
```
Comment (stefanloerwald) [inline Material.Blazor.Website/Pages/ChipsSelectMulti.razor:62]: Razor formatting issue?

File: Material.Blazor.Website/Pages/ChipsSelectSingle.razor [hunk 3]
```diff
@@ -60,7 +66,7 @@
         new MBIconBearingSelectElement<string> { SelectedValue = "brit-short", Label = "British Shorthair" },
         new MBIconBearingSelectElement<string> { SelectedValue = "russ-blue", Label = "Russian Blue", Icon="admin_panel_settings" },
         new MBIconBearingSelectElement<string> { SelectedValue = "ice-invis", Label = "Icelandic Invisible", Icon="card_membership" }
-    };
```
Comment (stefanloerwald) [inline Material.Blazor.Website/Pages/ChipsSelectSingle.razor:63]: razor formatting issue?

**Review by simonziegler — APPROVED** [review]
Carefully checked all updates.

**Pull request merged**